RecoTracker/TkDetLayers/src/TBLayer.h

```diff
@@ -25,14 +25,14 @@ class TBLayer: public BarrelDetLayer {
 
   // GeometricSearchDet interface
   
-  virtual const std::vector<const GeomDet*>& basicComponents() const final {return theBasicComps;}
+  virtual const std::vector<const GeomDet*>& basicComponents() const {return theBasicComps;}
```

Comment:
This specific line was not in plan, I think. I just forgot about cleaning up this lines.
The rest of the methods that are not final anymore, they should stay like that - for now, as Vincenzo said.